orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/knoxsso.xml.j2

```diff
@@ -127,9 +127,14 @@
             <value>-1</value>
         </param>
         <param>
+	   <!-- 
+	        This assumes that valid DNS is setup and access via a
+	        subdomain that ends in cloudera.com or cloudera.site.
+		If access is over IP the redirect will not match.
+           -->
            <name>knoxsso.redirect.whitelist.regex</name>
-           <value>.*</value>
+           <value>^\/.*$;^https?:\/\/(.+.cloudera.(?:com|site)):[0-9]+\/?.*$</value>
```

Comment:
In production it might be ok, it would work also on Mow *, once we register the endpoints to PEM service, bu we need a different approach in local:
- either use PEM locally
- put the pattern into java code and push down to Salt